language/documentation/hackathon-tutorial/step_3/BasicCoin.move

```diff
@@ -0,0 +1,22 @@
+module NamedAddr::BasicCoin {
+    struct Coin has store {
+        value: u64
+    }
+
+    struct Balance has key {
+        coin: Coin
+    }
+
+    /// Publish an empty balance resource under `account`'s address. This function must be called before
+    /// minting or transferring to the account.
+    public fun publish_balance(account: &signer);
```

Comment:
IIRC this is not correct Move syntax (no body with non-native functions). If so perhaps add a stub here `public fun publish_balance(account: &signer) { abort(1) }`?